### ISSUE TITLE
Error when using v2.0.0-rc.1 in IE 11

### ISSUE THREAD

**Issue by anschwa** [nuxt/vue-meta#371]
Hello,

My app will not load in IE 11 if I import the `vue-meta` package.
In the browser I get a white page and in the console I see an error ` SCRIPT1006: Expected ')'` in `app.js`

This was not an issue when previously using `vue-meta 1.6.0`, but I would like to use the release candidate because it finally solves the non-visible tab issue.
> The second Release Candidate fixes an issue introduced by the first RC and a long-standing issue when opening a page in a not-visible tab

**Comment by pimlie** [comment]
How are you using vue-meta? Do you have setup your transpiler correctly to target IE11?

Eg the error appears to originate from 
```
function _refresh(options = {}) {
```
as IE11 doesnt support default values for function arguments. But if I look at the build file that part is correctly transpiled to 
```
function _refresh(options) {
    if ( options === void 0 ) options = {};
```

**Comment by anschwa** [comment]
This is how my project is setup to hopefully target IE11 correctly. Does it look right to you?
```
# package.js
"browserslist": [
    "> 1%",
    "last 2 versions",
    "not ie <= 8"
  ],
"babel": {
  "presets": [
    [
      "@vue/app",
      {
        "useBuiltIns": "entry"
      }
    ]
  ]
}
```

```
# vue.config.js
transpileDependencies: ['vue-meta']
```

```
# main.js
import '@babel/polyfill';
import Meta from 'vue-meta';
...
Vue.use(Meta);
...
```

```
# App.vue
export default {
  name: 'app',
  metaInfo() {
    return {
      title: 'My App - ',
      titleTemplate: '%sMy App',
      link: [
        { rel: 'icon', href: this.iconLink },
      ],
    };
  },
  ...
}
  ```

Thanks!

**Comment by pimlie** [comment]
Looks good, but default babel config doesnt transpile anything from `node_modules` :)

You could either fix that or try to import the web-build of vue-meta (cause now you are importing the CommonJS build): `import Meta from 'vue-meta/lib/vue-meta'`

**Comment by anschwa** [comment]
@pimlie Importing from `vue-meta/lib/vue-meta` seems to work!

 It looked like changing the babel config to include `node_modules` would have worked as well, but I'm not sure which is the best approach to take. My understanding was that the `vue-cli` would take care of most of this package bundling on its own when building the app.
```
  "babel": {
    "include": ["/src/", "/node_modules/"],
    "exclude": [],
    "presets": [
      [
        "@vue/app",
        {
          "useBuiltIns": "entry"
        }
      ]
    ]
  }
```

**Comment by pimlie** [comment]
@anschwa As an update, actually this was caused by a faulty build (cjs/esm builds were not transpiled). It should be fixed in v2.0.2 so you dont need to use that specific import anymore.

**Comment by anschwa** [comment]
@pimlie Thanks for tracking that down!
